Cargo.lock

```diff
@@ -561,26 +561,26 @@ dependencies = [
 [[package]]
 name = "curve25519-dalek"
 version = "2.1.0"
-source = "git+https://github.com/garious/curve25519-dalek?rev=60efef3553d6bf3d7f3b09b5f97acd54d72529ff#60efef3553d6bf3d7f3b09b5f97acd54d72529ff"
```

Comment:
I couldn't figure out what was causing this file to automatically change for me. Any idea?